package.json

```diff
@@ -30,13 +27,11 @@
             "charts/"
         ]
     },
-    "resolutions": {
-        "@types/react": "17.0.0",
-        "vscode-languageserver-protocol": "3.15.3",
-        "vscode-languageserver-types": "3.15.1",
-        "react": "17.0.1",
-        "react-dom": "17.0.1",
-        "nan": "2.14.1",
-        "oniguruma": "7.2.1"
+    "dependencies": {
```

Comment:
why to add root dependencies?

Reply:
Hm, never touched this beyond removing `nan` and `oniguruma`. Maybe `yarn` switched from `resolutions` to `dependencies`? :thinking:

Reply:
I removed the section for now. Will double-check with Anton if there's something I'm missing here.